client/src/component/User.css

```diff
@@ -0,0 +1,32 @@
+.user-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .user-input {
+    margin-bottom: 10px;
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 14px;
+  }
+  
+  .user-input:focus {
+    outline: none;
+    border-color: #007bff;
+    box-shadow: 0 0 3px rgba(0, 123, 255, 0.5);
+  }
+  
+  .user-button {
+    background-color: #007bff;
+    color: #fff;
+    padding: 8px 16px;
+    border: none;
+    cursor: pointer;
+  }
+  
+  .user-button:hover {
+    background-color: #0056b3;
```

Comment:
nad here same as above , just two color same as figma

Reply:
Colors changed accoding to MVP